analysis/bin/vtsAnalyser.cc

```diff
@@ -324,34 +286,62 @@ void vtsAnalyser::MatchingForMC() {
     TLorentzVector jet_tlv;
     jet_tlv.SetPtEtaPhiM(Jet_pt[j], Jet_eta[j], Jet_phi[j], Jet_mass[j]);
     auto qIdx = tq1;
-    auto dr = tq1_tlv.DeltaR(jet_tlv); 
-    auto dr_ = tq2_tlv.DeltaR(jet_tlv); 
+    auto dr = tq1_tlv.DeltaR(jet_tlv);
```

Comment:
Everything from here to the next block should be simpler and more explicit. Its currently hard to see what you're trying to do

Reply:
Ok, I'll simplify here soon